CMakeLists.txt

```diff
@@ -166,15 +167,25 @@ endif()
 
 # external library: pybind11 (optional)
 if(openPMD_USE_PYTHON STREQUAL AUTO)
-    find_package(pybind11 2.2.1 CONFIG)  # 2.3.0
-    if(pybind11_FOUND)
+    if(openPMD_USE_INTERNAL_PYBIND11)
+        add_subdirectory("${openPMD_SOURCE_DIR}/share/openPMD/thirdParty/pybind11")
         set(openPMD_HAVE_PYTHON TRUE)
     else()
-        set(openPMD_HAVE_PYTHON FALSE)
+        find_package(pybind11 2.2.3 CONFIG)  # 2.3.0
```

Comment:
Do we require the changes from pybind/pybind11#1434?
If so, this is actually a lie.

Reply:
no, we don't. that's the beauty of it, see above :)